models/migrations/v188.go

```diff
@@ -0,0 +1,22 @@
+// Copyright 2021 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package migrations
+
+import (
+	"fmt"
+
+	"xorm.io/xorm"
+)
+
+func addOrgIDHookTaskColumn(x *xorm.Engine) error {
+	type HookTask struct {
+		OrgID int64 `xorm:"INDEX NOT NULL DEFAULT 0"`
+	}
+
+	if err := x.Sync2(new(HookTask)); err != nil {
+		return fmt.Errorf("Sync2: %v", err)
+	}
+	return nil
```

Comment:
But you should update the `org_id` from `webhook` table.

Reply:
Not needed, an orgnization event means an event which only related to an orgniztion (include `new team`, `new org member` ..), not means the events which related to any repos in a orgniztion, for them, just use repo id is enough and org_id should be set as zero, or ``FindOrgUndeliveredHookTasks()`` will get wrong result.